### PR TITLE
Added Ellen Spertus > names-generator.go

### PR DESCRIPTION
**Pull request by bojanstef**
Contributed to issue #17249

Signed-off-by: Bojan Stefanovic bojanstef4@gmail.com

### CODE REVIEW

**Comment by runcom** [comment]
@bojanstef thanks, could you pls squash your commits down to just one?

**Comment by thaJeztah** [comment]
Thanks @bojanstef, also please keep this in mind; https://github.com/bojanstef/docker/blob/17249-women-in-cs/pkg/namesgenerator/names-generator.go#L79

However, there's currently a PR open to add a men; https://github.com/docker/docker/pull/18296, so perhaps we can be okay if both are merged :innocent: 

@jfrazelle wdyt?

**Comment by jfrazelle** [comment]
sounds good to me just needs to be squashed

**Comment by bojanstef** [comment]
@jfrazelle - What is the proper steps for me to squash the previous commits, or is that already taken care of? I'm still learning 😊

Thanks all

**Comment by jfrazelle** [comment]
I would follow this
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
and then force push your branch after

On Mon, Nov 30, 2015 at 11:01 AM, Bojan Stefanovic <notifications@github.com

**Comment by thaJeztah** [comment]
Our contribution guide also has some information on rebasing, squashing (and a lot of useful stuff on contributing in general :+1: ) http://docs.docker.com/opensource/workflow/work-issue/#pull-and-rebase-frequently

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -159,6 +168,9 @@ var (
 		// Jane Colden - American botanist widely considered the first female American botanist - https://en.wikipedia.org/wiki/Jane_Colden
 		"colden",
 
+		// Lynn Conway - Computer scientist, electrical engineer, inventor, and transgenger activist. Co-author of "Introduction to VLSI Systems". - https://en.wikipedia.org/wiki/Lynn_Conway
+		"conway"
```
Comment (tiborvass) [inline pkg/namesgenerator/names-generator.go]: could also be John Conway

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -147,6 +153,9 @@ var (
 		// Walter Houser Brattain co-invented the transistor - https://en.wikipedia.org/wiki/Walter_Houser_Brattain
 		"brattain",
 
+		// Russian-American computer scientist and co-founder of Google with Larry Page. - https://en.wikipedia.org/wiki/Sergey_Brin
```
Comment (tiborvass) [inline pkg/namesgenerator/names-generator.go]: please respect the format: should start with name

File: pkg/namesgenerator/names-generator.go [hunk 3]
```diff
@@ -159,6 +168,9 @@ var (
 		// Jane Colden - American botanist widely considered the first female American botanist - https://en.wikipedia.org/wiki/Jane_Colden
 		"colden",
 
+		// Lynn Conway - Computer scientist, electrical engineer, inventor, and transgenger activist. Co-author of "Introduction to VLSI Systems". - https://en.wikipedia.org/wiki/Lynn_Conway
```
Comment (tiborvass) [inline pkg/namesgenerator/names-generator.go]: `transgender`

**Comment by tiborvass** [comment]
Also please respect gender parity, there are slightly more men than women added in this PR.
Also since Conway can be both Lynn and John, there is 2 men more than women.

**Comment by thaJeztah** [comment]
ping @bojanstef are you still working on this? can you address the comments above?

**Comment by thaJeztah** [comment]
@bojanstef I'll close this PR for now, but ping me if you still want to work on this, then I'll reopen so that you can push your changes

**Pull request closed without merging**